shared/src/types/admin/User.ts

```diff
@@ -10,30 +10,30 @@ export enum UserStatusKey {
 
 export type UserAddress = {
 	city?: string;
-	country?: string;
+	country?: string; // TODO: proper typing
 	number?: string;
 	street?: string;
-	zip?: string;
+	zip?: string; // TODO: proper typing (number?)
 };
 
 export type User = {
 	personal?: {
-		name?: string;
+		name?: string; // TODO: discuss if should be renamed to firstname
 		lastname?: string;
-		gender?: string;
+		gender?: string; // TODO: proper typing
 		company?: string;
 		phone?: string;
-		referral?: string;
+		referral?: string; // TODO: proper typing (enum?)
 	};
 	address?: UserAddress;
 	email: string;
 	status?: UserStatusKey;
 	stripe_customer_id?: string;
-	test_user?: boolean;
+	test_user?: boolean; // TODO: discuss if still needed
 	institution?: boolean;
-	language?: string;
-	location?: string;
-	currency?: string;
+	language?: string; // TODO: proper typing
+	location?: string; // TODO: discuss if still needed
```

Comment:
To be clarified with @ssandino.
At the moment, this is the most important location field for a user. This is set when a user goes through the donation process. `personal.address.country` is less often populated. Something to clarify!

Reply:
Yes, definitely. I wonder if we really need both...

Reply:
The address (incl. `country field`) is currently ONLY used for the donation certificate. I think it makes sense to keep a `country field` as part of the address entity. If we delete it, we run into new issues. For example: 

- Contributor has a foreign credit card, but lives currently Switzerland (i.e. a few from the staff). Kind of the expat situation. They might prefer using DE, UK, ZA as a main location (possible reasons: preferring to see EUR instead of CHF, or the future comparisons like home country to SL). But they still want a donation certificate which is sent to a Swiss address.
- Contributor from another country wants a certificate in the end of the year (we have this from Austria and Germany), as the tax authority "may" just accept a donation to a Swiss NGO or they need it for a matching with their company. 

That's why: we can't really say (a) the `country field` in an address is always _Switzerland_, (b) nor can we derive it from the `main location`. Therefore I would say: Let's keep it.